### DISCUSSION TITLE
Port to Android

### DISCUSSION THREAD

**Discussion by udlose** [tnmendes/watt#28] [Ideas]
Hi @tnmendes ,

I'm interested in porting this app to either MAUI.NET or Xamarin so it would be cross-platform functional. Would you have any objections? I would of course provide you credit for the original version. Thank you for your time.

Dave Black

**Comment by tnmendes** [comment]
Hey @udlose 
Thanks for the contact do you have availability for one of these days to make a video call?
Please send me an email to voless.app@gmail.com so we can schedule a conversation

**Comment by xflatcat** [comment]
Sounds a great idea.
